api/src/pcapi/core/operations/models.py

```diff
@@ -41,6 +43,13 @@ class SpecialEventQuestion(PcObject, Base, Model):
     title: str = sa.Column(sa.Text(), nullable=False)
 
 
+class SpecialEventResponseStatus(enum.Enum):
+    NEW = "new"
+    VALIDATED = "validated"
+    REJECTED = "rejected"
+    PRESELECTED = "preselected"
```

Comment:
Détail : Si on veut que ce soit plus homogène dans le format avec les `ValidationStatus` ou encore `BookingStatus` : 
```suggestion
    NEW = "NEW"
    VALIDATED = "VALIDATED"
    REJECTED = "REJECTED"
    PRESELECTED = "PRESELECTED"
```